Mon Mar 20 11:01:45 2017
Error generated by: Connect : init(frame)
Error description: Can't find a face

